mobile/src/main/res/values-de/strings.xml

```diff
@@ -15,76 +15,79 @@
-    <string name="settings_openhab_alturl">openHAB Remote URL</string>
-    <string name="settings_openhab_alturl_summary">Remote openHAB URL (genutzt, wenn openHAB lokal nicht gefunden wird)</string>
+    <string name="settings_openhab_url">openHAB-URL</string>
+    <string name="settings_openhab_url_summary">URL des openHAB-Servers. Wenn gesetzt, ist Auto-Discovery deaktiviert.</string>
+    <string name="settings_openhab_alturl">openHAB-Remote-URL</string>
+    <string name="settings_openhab_alturl_summary">URL eines openHAB-Servers außerhalb des lokalen Netzwerkes. Wird genutzt, wenn im lokalen Netzwerk kein openHAB-Server gefunden wurde.</string>
     <string name="settings_openhab_username">Benutzername</string>
-    <string name="settings_openhab_username_summary">openHAB Benutzername</string>
+    <string name="settings_openhab_username_summary">openHAB-Benutzername</string>
     <string name="settings_openhab_password">Passwort</string>
-    <string name="settings_openhab_password_summary">openHAB Passwort</string>
-    <string name="settings_openhab_sslclientcert">SSL Client Zertifikat</string>
-    <string name="settings_openhab_sslclientcert_howto">SSL Client Zertifikat How-To</string>
-    <string name="settings_openhab_screentimeroff">Deaktiviere Ruhezustand</string>
-    <string name="settings_openhab_screentimeroff_summary">Schaltet das automatische Aktivieren des Ruhezustands nach Inaktivität ab, wenn die openHAB App läuft.</string>
-    <string name="settings_openhab_demomode">Demo Modus</string>
-    <string name="settings_openhab_demomode_summary">Betreibt openHAB im Demo Modus</string>
-    <string name="settings_openhab_theme">openHAB Thema</string>
-    <string name="settings_openhab_appversion">openHAB App Version</string>
-    <string name="settings_openhab_sslcert">Ignoriere SSL Zertifikat</string>
-    <string name="settings_openhab_sslcert_summary">Ignoriere Zertifikatsvalidierung beim SSL-Handshake</string>
-    <string name="settings_openhab_sslhost">Ignoriere SSL Hostnamen</string>
-    <string name="settings_openhab_sslhost_summary">Ignoriere Hostname-Prüfung beim SSL-Handshake</string>
+    <string name="settings_openhab_password_summary">openHAB-Passwort</string>
+    <string name="settings_openhab_sslclientcert">SSL-Client-Zertifikat</string>
+    <string name="settings_openhab_sslclientcert_howto">Erläuterungen zum SSL-Client-Zertifikat</string>
+    <string name="settings_openhab_screentimeroff">Ruhezustand deaktivieren</string>
+    <string name="settings_openhab_screentimeroff_summary">Schaltet das automatische Aktivieren des Ruhezustands nach Inaktivität ab, wenn die openHAB-App läuft</string>
+    <string name="settings_openhab_demomode">Demo-Modus</string>
+    <string name="settings_openhab_demomode_summary">Demo-Daten an Stelle von realen Server-Daten verwenden</string>
+    <string name="settings_openhab_theme">App-Design</string>
```

Comment:
Let's get some more opinions: https://forum.f-droid.org/t/how-to-translate-theme/853